### ISSUE TITLE
"safe" drones can still pickpocket

### ISSUE THREAD

**Issue by ArcLumin** [HippieStationArchive/HippieStation13-1#3045]
wew

**Comment by Swagile** [comment]
i always knew drones were secretly gremlins in disguise

this is just proof, i tell you

**Comment by ghost** [comment]
Yes, because pockets are snowflake code and don't work with the normal strip menu. I can make them not be able to strip but it's snowflakey as hell.

**Comment by ghost** [comment]
https://github.com/McBawbaggings/HippieStation13/blob/7323ddf15810b28712b5082e8e7dc8c45f0d9e2f/code/modules/mob/living/carbon/human/human.dm#L424
